environment.yml

```diff
@@ -5,9 +5,7 @@ name: improver
 channels:
   - conda-forge
 dependencies:
-  - python=3
+  - python=3.6
```

Comment:
Gavin, I don't believe that the environment file is only there for read the docs. As such I think we need to keep the dependencies in here and instead add this filtered environment into a separate env file, e.g. doc/rtd_environment.yaml.

Might be worth checking with Ben/Aaron as I could be wrong about the need to keep this complete.

Reply:
iirc I think it only is here for readthedocs - there's a separate one for Travis, and I don't think this is being picked up elsewhere. I'd be happy if we wanted to rename this file, but then we'd need to point to it with the `.readthedocs.yml` file.